charts/spire/charts/spiffe-oidc-discovery-provider/templates/deployment.yaml

```diff
@@ -96,7 +99,7 @@ spec:
           image: {{ template "spiffe-oidc-discovery-provider.image" .Values.telemetry.prometheus.nginxExporter }}
           imagePullPolicy: {{ .Values.telemetry.prometheus.nginxExporter.image.pullPolicy }}
           args:
-          - -nginx.scrape-uri=http://127.0.0.1/stub_status
+          - -nginx.scrape-uri=http://127.0.0.1:8080/stub_status
```

Comment:
Can we do this in a prometheus/metrics PR?

Reply:
Not really prom related. The pr here is switching to unprivileged nginx, so it cant run at port 80 so it now runs at port 8080. The exporter then breaks as its hardcoded to assume port 80 and needs updating too. I'd prefer not to break something in one pr and fix it in another.